metadata-ingestion/src/datahub/ingestion/source/mlflow.py

```diff
@@ -165,6 +173,13 @@ def __init__(self, ctx: PipelineContext, config: MLflowConfig):
             tracking_uri=self.config.tracking_uri,
             registry_uri=self.config.registry_uri,
         )
+        self._configure_auth()
+
+    def _configure_auth(self):
+        """Configure MLflow authentication based on config"""
+        if self.config.username and self.config.password:
+            os.environ["MLFLOW_TRACKING_USERNAME"] = self.config.username
+            os.environ["MLFLOW_TRACKING_PASSWORD"] = self.config.password
```

Comment:
can we pass these into the `MlflowClient` directly?

Reply:
e.g. use https://mlflow.org/docs/latest/api_reference/auth/python-api.html#mlflow.server.auth or something

Reply:
I've tried this approach but seems like it's not very straightforward -- AFAIK  we set perm on every single model/experiment in this way